### ISSUE TITLE
0.3.12 tag instead of v0.3.12

### ISSUE THREAD

**Issue by yhrn** [imdario/mergo#181]
The current latest release is tagged `0.3.12` instead of `v0.3.12` which means the module version becomes ugly:

```
> go get github.com/imdario/mergo@v0.3.12
go get: github.com/imdario/mergo@v0.3.12: invalid version: unknown revision v0.3.12
> go get github.com/imdario/mergo@0.3.12 
go: downloading github.com/imdario/mergo v0.3.12-0.20210303221052-29fb3d3bdc55
go get: upgraded github.com/imdario/mergo v0.3.11 => v0.3.12-0.20210303221052-29fb3d3bdc55
```

**Comment by smgladkovskiy** [comment]
@imdario please!

**Comment by imdario** [comment]
Ouch, sorry!

**Comment by imdario** [comment]
@yhrn @smgladkovskiy Re-released as v0.3.12.
